### PR TITLE
Enable OnPush ChangeDetection in SvgKeyboardWrapComponent

### PR DESCRIPTION
**Pull request by fjozsef**
I had to downgrade the angular to the latest stable, because there was an error in the animation with the latest beta. (I couldn't find out if the bug is at our side or at their side.)

### CODE REVIEW

File: src/components/svg/wrap/svg-keyboard-wrap.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template ngIf="layers">
     <layers [class.disabled]="popoverShown" (select)="selectLayer($event.oldIndex, $event.index)" [current]="currentLayer"></layers>
     <div class="keyboard-slider" (mouseout)="hideTooltip($event)">
-        <svg-keyboard *ngFor="let layer of layers"
+        <svg-keyboard *ngFor="let layer of layers; trackBy: trackKeyboard"
```
Comment (NejcZdovc) [inline src/components/svg/wrap/svg-keyboard-wrap.component.html]: Why would be this used here?

Comment (fjozsef) [inline src/components/svg/wrap/svg-keyboard-wrap.component.html]: Otherwise the angular would instantiate a new svg-keyboard, now it calls OnChanges,

Comment (NejcZdovc) [inline src/components/svg/wrap/svg-keyboard-wrap.component.html]: Ok

File: src/store/reducers/keymap.ts [hunk 2]
```diff
@@ -99,11 +101,29 @@ export default function(state = initialState, action: Action): KeymapState {
...
+            changedKeymap.layers[layerIndex] = changedLayer;
+
+            const moduleIndex: number = action.payload.module;
+            const module: Module = changedLayer.modules[moduleIndex];
+            const changedModule: Module = new Module();
+            Object.assign(changedModule, module);
+            changedLayer.modules[moduleIndex] = changedModule;
+
+            const keyIndex: number = action.payload.key;
+            changedModule.keyActions[keyIndex] = action.payload.keyAction;
```
Comment (NejcZdovc) [inline src/store/reducers/keymap.ts]: I think that here should be a new constructor copy

Comment (fjozsef) [inline src/store/reducers/keymap.ts]: Fixed

**Pull request merged**